FWCore/Framework/test/stubs/ToyAnalyzers.cc

```diff
@@ -95,6 +96,75 @@ namespace edmtest {
     std::vector<edm::EDGetTokenT<IntProduct>> m_tokens;
   };
 
+  //--------------------------------------------------------------------
+  //
+  template <typename T>
+  class GenericIntsAnalyzerT : public edm::global::EDAnalyzer<> {
```

Comment:
Does this really apply to only Ints?

Reply:
Good catch, it could be just `GenericAnalyzerT`.

Reply:
Renamed to `GenericAnalyzerT`.